django/applications/catmaid/control/skeletonexport.py

```diff
@@ -1804,7 +1806,8 @@ def connector_polyadicity(request:HttpRequest, project_id=None) -> JsonResponse:
         ''')
 
     cursor = connection.cursor()
-    cursor.execute('''
+    cursor.execute(
+        '''
```

Comment:
How comes this is changed here to a less compact format while kept in most other places (like a few lines above)? While I think I prefer the more compact form, I am fine with either version, but would generally try to strive for consistency.

Reply:
Yes, this could be changed back - some of the indentation flake8 rules aren't very clear so I thought this was necessary to fix the error, but it turns out it isn't.

Reply:
It looks like in some cases this form is actually needed to prevent triggering

    E123 closing bracket does not match indentation of opening bracket's line

Whether or not that happens may depend on whether the string has format arguments, or whether there are subsequent arguments to the `execute` method.

Reply:
But then the previous format should have worked, shouldn't it? So far most of the `cursor.execute()` blocks follow this pattern:

```
cursor.execute("""
    SQL
""", {
    "parameter_key": "value",
})
```

or, if additional `format` operations are needed for the query (like in this example):

```
cursor.execute("""
    SQL
""".format(**{
    "parameter_key": "value",
}), {
    "parameter_key": "value",
})
```

There are probably outliers, but overall this seems to be be used consistently in most places and it is pretty readable to me. I am fine with replacing the `format()` with f-strings though. If not, I like the `**` operator, because it makes the parameterization of both parts consistently use dicts, which is easier to read for me.

How should the example queries above look like according to `flake8`? From your changes I assume like this (?):

```
cursor.execute(
    """
    SQL
    """.format(**{
        "parameter_key": "value",
    }), {
        "parameter_key": "value",
    })
```

I agree though that further changes here shouldn't be part of this PR. Reading through the current set of changes took already long enough. :-)

Reply:
> I am fine with replacing the format() with f-strings though.

Happy for this to be the goal in the future.

> If not, I like the ** operator, because it makes the parameterization of both parts consistently use dicts, which is easier to read for me.

Personal taste, I suppose; I prefer `a="b", c="d"` to `**{"a": "b", "c": "d"}`, the latter adds a lot of characters which don't add any meaning. I didn't think about the attraction of using dicts in both cases.

> How should the example queries above look like according to flake8?

Both of those are actually fine, I think - I suspect a couple of of them originally may have had some other funky indentation and I got carried away in fixing that. Personally I prefer that middle lines do not return to the same indentation as the enclosing parentheses, and it may have been an interaction with that preference that led to the format changes I made. I'll see about reverting this.